lisa/environment.py

```diff
@@ -362,7 +365,7 @@ def _reset(self) -> None:
         self.runbook.reload_requirements()
 
         self.is_new = True
-
+        self._information_cache = {}
```

Comment:
Use `None`, so if nothing is found, it doesn't update every time.

Reply:
Updated